R/shiny-recorder.R

```diff
@@ -215,6 +215,10 @@ RecordingSession <- R6::R6Class("RecordingSession",
         httpuv::interrupt()
         private$localServer <- NULL
         close(private$outputFile)
+        if (length(fileList)) {
+          fileList <- paste(sprintf("\t%s", private$postFiles), collapse = '\n')
```

Comment:
Line 219 needs to be above line 218. (`fileList` does not exist in 218)

Reply:
Ah yes, that first occurrence of `fileList` actually needed to be `private$postFiles`